lib/awful/menu.lua

```diff
@@ -443,6 +465,7 @@ function menu:add(args, index)
         return
     end
     item.parent = self
+    item.enabled = item.enabled == nil and true or item.enabled
```

Comment:
The `item.enable` property is supposed to be a non-null boolean, based on the `menu.entry` constructor implementation below. So I would remove this extra test and keep `item.enable` as-is.